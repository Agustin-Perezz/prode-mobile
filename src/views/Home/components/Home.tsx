import { Text, View } from 'react-native';

export const Home = () => (
  <View>
    <Text>Home View.</Text>
  </View>
);
