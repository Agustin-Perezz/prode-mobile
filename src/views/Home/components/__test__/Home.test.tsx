import React from 'react';
import renderer from 'react-test-renderer';

import { Home } from '../Home';

describe('<Home />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
