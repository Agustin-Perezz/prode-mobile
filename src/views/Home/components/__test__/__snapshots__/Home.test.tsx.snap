// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should match snapshot 1`] = `
<View>
  <Text>
    Home View.
  </Text>
</View>
`;
